src/components/LinkChooser/mock/guidelines.ts

```diff
@@ -0,0 +1,51 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconLabel } from "../types";
+
+export const guidelines = [
+    {
+        id: "15",
+        preview:
+            "https://images.frontify.test/s3/frontify-dev-files/eyJwYXRoIjoibXNpcmljXC9hY2NvdW50c1wvYzRcLzFcL3Byb2plY3RzXC8yXC9hc3NldHNcLzhmXC83XC85MWU4MzdjZDk2YWQzN2EwYWY4OTZhYzA5OWRlNzIyNi0xNjM0MzAxNDUxLmpwZyJ9:msiric:Li3v_e6ogrxBoH-kli3whbqekW1K4ar5Dshif80q2o0?width=2400&height={height}",
```

Comment:
`images.frontify.test` is not working outside of your local env

Reply:
fixed